tests/stress_tests/scripts/get_testdata.py

```diff
@@ -148,10 +149,12 @@ def main():
     cmd = '{downloader_path} --list {models_list_path}' \
           ' --num_attempts {num_attempts}' \
           ' --output_dir {models_dir}' \
-          ' --cache_dir {cache_dir}'.format(downloader_path=downloader_path, models_list_path=models_list_path,
-                                            num_attempts=OMZ_NUM_ATTEMPTS,
-                                            models_dir=args.omz_models_out_dir,
-                                            cache_dir=args.omz_cache_dir)
+          ' --cache_dir {cache_dir}' \
+          ' --jobs {jobs_num}'.format(downloader_path=downloader_path, models_list_path=models_list_path,
```

Comment:
@asomsiko downloader has an opportunity to parallel downloading (and retrieving from cache). It's POC, perhaps we want to specify --download_jobs and --convert_jobs within cmd (I think it shouldn't be equal values)

Reply:
YAGNI